app/src/main/java/io/apicurio/registry/content/ContentHandleCustomizer.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.content;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.jsontype.TypeSerializer;
+import com.fasterxml.jackson.databind.module.SimpleSerializers;
+import com.fasterxml.jackson.databind.ser.Serializers;
+import io.quarkus.jackson.ObjectMapperCustomizer;
+
+import java.io.IOException;
+import java.util.Collections;
+import javax.inject.Singleton;
+
+/**
+ * @author Ales Justin
+ */
+@Singleton
+public class ContentHandleCustomizer implements ObjectMapperCustomizer {
```

Comment:
Is this actually used right now?  Where do we have a `ContentHandler` as a property in a greater JSON document?  Does this happen somewhere in the confluent compatibility api impl?

Reply:
No, w/o this, plain Response doesn't know how to handle any ContentHandle type.
e.g. we return ContentHandle instance in any Response.ok(artifact.content)

Reply:
OK I assumed that the `ContentHandleMessageBodyWriter` class handled the `Response.ok(artifact.content)` case, but you're saying that this object mapper customizer does it instead?

Reply:
Ah, yes, we could drop that CHCustomizer, and just have this CHMBW ... 
Let me check if that works as expected ... since I think it's different if you write bytes vs raw text ...